RadixWallet/Features/AssetTransferFeature/AssetTransfer+Reducer.swift

```diff
@@ -114,6 +114,9 @@ extension AssetTransfer.State {
 			guard $0.recipient != nil else {
 				return false
 			}
+			guard $0.isDepositEnabled else {
```

Comment:
I think we should only use unnamed arguments in very small closures

Reply:
I know you didn't start it, but might as well take the opportunity to fix it

Reply:
will do!